index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html ng-app="metroMusuem">
+<head>
+  <meta charset="UTF-8">
+  <title>The Metropolitan Museum of Art</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+  <script src="app.js"></script>
+  <script src="js/comingup.js"></script>
+  <script src="js/generalinfo.js"></script>
+  <script src="js/donor.js"></script>
+</head>
+<body>
+  <h1>The Metropolitan Museum of Art</h1>
+  <section ng-controller="ComingUpController as coming">
+    <h1>Exhibit:</h1>
+    <h2>{{ coming.title }}</h2>
+    <p>By {{ coming.artist }}</p>
+  </section>
+  <hr>
+  <section ng-controller="GeneralInfoController as general">
+    <h1>General Info:</h1>
+    <h2>Hours:</h2>
+    <ul>
+      <li ng-repeat="(key, value) in general.hours"><b>{{key}}:</b>{{value}}</li>
+    </ul>
+    <h2>Suggested Admission:</h2>
+    <ul>
+      <li ng-repeat="(key, value) in general.admission">
+        <b>{{key}}:</b> ${{value}}
+      </li>
+    </ul>
+
+  </section>
+  <hr>
+  <section ng-controller="DonarController as donor">
+    <h1>Donor Information:</h1>
+    <h2>Contacts:</h2>
+    <p><a href={{ donor.contact.email }}>{{ donor.contact.email }}</a> - {{ donor.contact.phone }}</p>
```

Comment:
If you link email, you could add [mailto](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Email_links) :)
```html
<a href="mailto:{{ donor.contact.email }}">
```